jekyll/_config.yml

```
@@ -68,7 +68,7 @@ asciidoc:
       terraformversion: 0.13.0
       kubectlversion: 1.19
       helmversion: 3.4.0
-      kotsversion: 1.41
+      kotsversion: 1.43
```

Version must be greater than 1.43 I believe. Can confirm that 1.44 worked for Pawan who has faced the original issue.

```suggestion
      kotsversion: 1.44
```